modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/TracksAnalyticsTracker.kt

```diff
@@ -81,7 +81,7 @@ class TracksAnalyticsTracker @Inject constructor(
             PredefinedEventProperty.HAS_DYNAMIC_FONT_SIZE to displayUtil.hasDynamicFontSize(),
             PredefinedEventProperty.USER_IS_LOGGED_IN to isLoggedIn,
             PredefinedEventProperty.PLUS_HAS_SUBSCRIPTION to hasSubscription,
-            PredefinedEventProperty.PLUS_HAS_LIFETIME to hasLifetime,
+            PredefinedEventProperty.PLUS_HAS_LIFETIME to isPocketCastsChampion,
```

Comment:
We agreed that we are not going to update this PLUS_HAS_LIFETIME event name. See p1721917283521179/1720493861.494549-slack-C05RR9P9RAT